library.properties

```diff
@@ -2,9 +2,9 @@ name=Arduino_ConnectionHandler
 version=0.9.0
 author=Ubi de Feo, Cristian Maglie, Andrea Catozzi, Alexander Entinger et al.
 maintainer=Arduino <info@arduino.cc>
-sentence=Arduino Library for network connection management (WiFi, GSM, NB, [Ethernet])
+sentence=Arduino Library for network connection management (WiFi, GSM, NB, [Ethernet], Notecard)
 paragraph=Originally part of ArduinoIoTCloud
 category=Communication
 url=https://github.com/arduino-libraries/Arduino_ConnectionHandler
-architectures=samd,esp32,esp8266,mbed,megaavr,mbed_nano,mbed_portenta,mbed_nicla,mbed_opta,mbed_giga,renesas_portenta,renesas_uno,mbed_edge
-depends=Arduino_DebugUtils, WiFi101, WiFiNINA, MKRGSM, MKRNB, MKRWAN
+architectures=samd,esp32,esp8266,mbed,megaavr,mbed_nano,mbed_portenta,mbed_nicla,mbed_opta,mbed_giga,renesas_portenta,renesas_uno,mbed_edge,stm32
+depends=Arduino_DebugUtils, Blues Wireless Notecard (>=1.6.0), WiFi101, WiFiNINA, MKRGSM, MKRNB, MKRWAN
```

Comment:
It would be better to not change the library dependency order, please add  Blues Wireless Notecard at the end of the list

Reply:
No problem. I didn't know that it was important, so I was just putting it in alphabetical order.